anax-chrome/src/main/java/org/anax/framework/config/AnaxChromeDriver.java

```diff
@@ -71,7 +71,7 @@ public AnaxDriver getWebDriver(@Value("${anax.localdriver:true}") Boolean useLoc
     @ConditionalOnMissingBean
     @Bean
     public WebController getWebController(@Autowired AnaxDriver anaxDriver, @Value("${anax.defaultWaitSeconds:5}") Integer defaultWaitSeconds) throws Exception {
-        return new WebDriverWebController(anaxDriver.getWebDriver(), defaultWaitSeconds);
+        return new WebDriverWebController(anaxDriver.getWebDriver(), anaxDriver, defaultWaitSeconds);
```

Comment:
@dimitrianos pls simplify the signature to `new WebDriverWebController(anaxDriver, defaultWaitSeconds)` since you can receive the WebDriver from that object - no need to have both of them.